### PR TITLE
Tc01

### PR DESCRIPTION
**Pull request by cvan8397**

### CODE REVIEW

File: src/main/java/models/Account.java [hunk 1]
```diff
@@ -1,17 +1,10 @@
...
     private String password;
-
-    public String getDescription() {
-        return description;
-    }
-
-    public void setDescription(String description) {
-        this.description = description;
-    }
+    private String confirmPassword;
```
Comment (LoliSlayerMC) [inline src/main/java/models/Account.java]: confirmPassword is not a valid property

File: src/test/java/tests/TestBase.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import helper.Constant;
+import helper.Log;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public class TestBase {
+    @BeforeMethod
+    public void setup() {
+        Log.info("Pre-condition");
+        BrowserHelper.startBrowser(BrowserHelper.DriverType.CHROME);
```
Comment (LoliSlayerMC) [inline src/test/java/tests/TestBase.java]: find another way to pass driver, shouldn't hardcode chrome

**Review by LoliSlayerMC — CHANGES_REQUESTED** [review]

**Comment by cvan8397** [comment]
@LoliSlayerMC I have fixed my code based on your request. Please review and let me know if any problems

**Review by LoliSlayerMC — APPROVED** [review]
LGTM

**Pull request merged**